porter/v1/cluster_control_plane.proto

```diff
@@ -196,6 +196,16 @@ message ApplyPorterAppResponse {
   EnumCLIAction cli_action = 2;
 }
 
+message UpdateFailedRevisionRequest {
```

Comment:
Can we call this `UpdateRevisionStatus` and have it potentially handle many kinds of status updates? i.e. not just failed